packages/libs/react-ui/src/components/Form/Input/InputCopy/InputCopy.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import type { IInputProps } from '@components/Form';
+import { InputCopy } from '@components/Form';
+
+import type { SystemIcon } from '@components/Icon';
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+const meta: Meta<IInputProps> = {
+  title: 'Form/InputCopy',
```

Comment:
Could we add another folder to group all input stories together? I think you can just update this to 'Form/Input/InputCopy'